### DISCUSSION TITLE
Scaling Error and cutting issue

### DISCUSSION THREAD

**Discussion by KingDWS** [arkypita/LaserGRBL#1297] [Q&A]
I am using v3.8.5.  I generated a shape saved as a dxf as well as jpg. Both import just fine. The final burned size should be 145mm but when I set it to that size it comes out at 72mm. I can set it to autosize or specify the size and it still does not come out at 146.
The only way I get it to come out to the right size is by setting $100 and $101 to 146. I did a calibration test today out of curiosity and a 200mm line in X and Y were bang on at 200. I am just starting to use LaserGrbl on a new laser so am not that familiar with it. So how or what do I do to get the burned dimensions correct?

Also earlier I did a small test shape over and over to get the right power and speed settings. It was say 40mmx10mm and came out perfectly. However when I just go to a larger shape the quality drops to having maybe 60% of the figure cut and the  edge fairly rough. Any idea what might have caused that? Acceleration setting? 

I was starting off with the small size to be able to quickly get the settings then went to a medium size shape ie the 145mm thing before trying the big final shape that is 950mm long. I had thought that once I had the settings for the material I could simply burn any size shape. So I am missing something here but no idea what. Appreciate any help!

Thanks

Dave

**Comment by arkypita** [comment]
Hi

If the difference is between what you set as size in "import size and option form", and the preview given by LaserGRBL window, this may be because you have white space near your subject in the original image. Please use a graphic editor program (gimp/photoshop/paint) to crop the subject of your image.

If you are doing a vector import (svg... dxf is not supported yet) maybe you have some offset/white space between the object and the "page". Please use a vector editor program (inkscape) to move your subject near 0,0 of your page.

If the difference is between what you see in LaserGRBL preview and what you have in engraving result, please configure your grbl configuration.

https://lasergrbl.com/faq/
